modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/table-view/DropZone.es.js

```diff
@@ -46,7 +46,7 @@ const generateItems = (columns, rows = 10) => {
 
 const DropZone = ({fields, onAddFieldName, onRemoveFieldName}) => {
 	const [{canDrop, overTarget}, drop] = useDrop({
-		accept: DragTypes.DRAG_FIELD_TYPE,
```

Comment:
@diegonvs can you check if we should change here too? I think this is for another screen that is not related to Form Builder, just to check, try to create a Form View and Table View.

Reply:
Nope, I'll not change it. Table View is currently using EditorVariant's useDrop and the useDrag from the FieldType component. When hard-coding `fieldType` it won't work